app/assets/javascripts/student_profile/record_service.jsx

```diff
@@ -66,20 +67,34 @@ import serviceColor from '../student_profile/service_color.js';
       const {nowMoment} = this.props;
       return {
         serviceTypeId: null,
-        providedByEducatorName: "",
+        providedByEducatorName: ''  ,
         dateStartedText: nowMoment.format('MM/DD/YYYY'),
-        estimatedEndDateText: nowMoment.format('MM/DD/YYYY')
+        estimatedEndDateText: ''
```

Comment:
@snoopyuri suggested over email to change this to "the end of the current school year". We should do that, and this PR doesn't reflect that yet.